tests/functional/helpers.js

```diff
@@ -1,8 +1,8 @@
 const checkPeopleInList = (peopleExpected, listLabel) => {
-  $(listLabel(1)).waitForDisplayed();
+  $(listLabel(0)).waitForDisplayed();
 
   for (let i = 1; i <= peopleExpected.length; i++) {
-    expect($(listLabel(i)).getText()).to.equal(peopleExpected[i - 1]);
+    expect($(listLabel(i - 1)).getText()).to.equal(peopleExpected[i - 1]);
```

Comment:
ids and data-qa attributes for multiple elements should all start at an index of 1.